index.html

```diff
@@ -21,7 +21,7 @@
     <link rel="stylesheet" href="style4.css">
     <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
     integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
-    <title>Mugisha's Portfolio</title>
+    <title>Mugisha's  Portfolio</title>
```

Comment:
- [OPTIONAL] Kindly make sure your design matches the template design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) for better user-interface(UI)
- Figma design
![screenshot (2)](https://user-images.githubusercontent.com/44624138/145797812-cf73b474-06e2-48b7-b8d6-740aa203ff76.png)
![screenshot (1)](https://user-images.githubusercontent.com/44624138/145797753-675ca60a-d54f-48dc-a6ab-d8b5271515e3.png)

- Your design
![screenshot (3)](https://user-images.githubusercontent.com/44624138/145797899-15692f62-2aff-4006-9106-a3db61b7a53c.png)
![screenshot (4)](https://user-images.githubusercontent.com/44624138/145797918-661fbce1-1172-4f3d-8cea-7d2517f7242f.png)